### PR TITLE
{Joshua} - Sage - An Introduction to Layer 1 and Layer 2 Protocols

### PR DESCRIPTION
**Pull request by jorshimayor**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [x] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [x] The title of the Pull Request matches the title of my tutorial.
- [x] I've followed the tutorial template structure.
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [ ] I've thoroughly explained the code, with inline code comments and text explanations.
- [ ] I've run a linter & prettifier to format my code blocks.
- [ ] I've run the tutorial to test it and the code works.

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

File: blog/13-12-22-layer-1-and-layer-2/Layer-1-and-layer-2.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+---
+title: An Introduction to Layer 1 and Layer 2 Blockchain Protocols
+description: In this tutorial, we will provide an introduction to the different types of blockchain protocols and explain the key differences between layer 1 and layer 2 protocols. We will discuss the benefits and limitations of each type of protocol, as well as provide examples of how they are used in the real world. By the end of this tutorial, you will have a better understanding of the different types of blockchain protocols and how they are used in web3 development. This tutorial is essential for anyone looking to learn about blockchain protocols and how they impact the development of decentralized applications.
```
Comment (viral-sangani) [inline blog/13-12-22-layer-1-and-layer-2/Layer-1-and-layer-2.md]: Please reduce the description to 1-2 lines.

File: blog/13-12-22-layer-1-and-layer-2/Layer-1-and-layer-2.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+authors:
+  - name: ✍️ Obafemi Joshua 
+tags: [celo]
+hide_table_of_contents: true
+slug: "/tutorials/An-Introduction-to-Layer-1-and-Layer-2-Blockchain-Protocols"
+---
+
+
+#Layer 1 and Layer 2 Protocols.
+blog\13-12-22-layer-1-and-layer-2\images\img1 .png
```
Comment (viral-sangani) [inline blog/13-12-22-layer-1-and-layer-2/Layer-1-and-layer-2.md]: Please add the image in markdown format.
https://marinegeo.github.io/2018-08-10-adding-images-markdown/

File: blog/13-12-22-layer-1-and-layer-2/Layer-1-and-layer-2.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+ Image by Crypto Demystified
+
+Meta Description: Layer 1 and Layer 2 protocols uniquely determine blockchain scalability, which seems to improve as you progress from base structure Layer 1 to Layer 2. Learn more!
+
+#Introduction
```
Comment (viral-sangani) [inline blog/13-12-22-layer-1-and-layer-2/Layer-1-and-layer-2.md]:
```suggestion
## Introduction
```

File: blog/13-12-22-layer-1-and-layer-2/Layer-1-and-layer-2.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+We can't talk of blockchain solutions without mentioning Layer 1 and Layer 2 protocols and how they uniquely contribute to blockchain scalability. It is essential to know that Layer 1 primarily serves as the framework for Layer 2 and its solutions. Without Layer 1 scaling solutions, the consensus protocol, sharding, and the lightning network would not be possible.
+
+
+#What are Layer 1 Blockchains?
```
Comment (viral-sangani) [inline blog/13-12-22-layer-1-and-layer-2/Layer-1-and-layer-2.md]:
```suggestion
## What are Layer 1 Blockchains?
```

File: blog/13-12-22-layer-1-and-layer-2/Layer-1-and-layer-2.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+By definition, a protocol is Layer 1 when it completes transactions on its own blockchain. It's the term used for describing the underlying principal blockchain architecture. An example of a Layer 1 blockchain is the Celo blockchain which was branched from Go Ethereum (Geth) in 2017. It has, however, made some substantial adjustments, such as putting PoS and a distinctive address system in place. 
+The Celo ecosystem, which comprises of NFTs, payment methods, and DeFi, has over 100 million confirmed transactions. Also, anyone can create a public key using a phone number or email address. There is no need for specialist hardware since standard PCs can operate the blockchain with ease. 
+
+#What are Layer 2 Blockchains?
```
Comment (viral-sangani) [inline blog/13-12-22-layer-1-and-layer-2/Layer-1-and-layer-2.md]:
```suggestion
## What are Layer 2 Blockchains?
```

File: blog/13-12-22-layer-1-and-layer-2/Layer-1-and-layer-2.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+Alternatively, a Layer 2 protocol is one that is built on top of an existing blockchain. The primary goal of this protocol is to address the major cryptocurrency networks' transaction speed and scaling issues. 
+Without a doubt, Layer 1 and Layer 2 protocols uniquely determine blockchain scalability, which seems to improve as you progress from base structure Layer 1 to Layer 2. But why is scalability a complex blockchain problem that needs immediate solutions?
+
+#Why is Blockchain Scalability Important?
```
Comment (viral-sangani) [inline blog/13-12-22-layer-1-and-layer-2/Layer-1-and-layer-2.md]:
```suggestion
## Why is Blockchain Scalability Important?
```

**Review by viral-sangani — APPROVED** [review]

**Comment by jorshimayor** [comment]
Hello, 

Kindly check if the changes are correct thanks

**Pull request merged**